pkg/driver/sqlite/sqlite_test.go

```diff
@@ -405,3 +405,18 @@ func TestSQLiteQuotedMigrationsTableName(t *testing.T) {
 		require.Equal(t, `"fooMigrations"`, name)
 	})
 }
+
+func TestSQLiteFTS5MigrationsTable(t *testing.T) {
```

Comment:
As this test really shouldn't duplicate testing migration functionality, the name should be updated to reflect that.

I propose `TestSQLiteFTS5Available` as the test name, as that is what it is testing for.

Reply:
Agreed. Renamed.